Makefile

```diff
@@ -186,14 +186,8 @@ list_test_data_options:
 	gsutil ls $(REGRESSION_DATA_STORAGE_BUCKET)/$(FORTRAN_SERIALIZED_DATA_VERSION)
 
 lint:
-	pre-commit run
-	# pre-commit runs black for now. Will also run flake8 eventually.
-	# black --diff --check $(PYTHON_FILES) $(PYTHON_INIT_FILES)
-	# disable flake8 tests for now, re-enable when dycore is "running"
-	#@flake8 $(PYTHON_FILES)
-	# ignore unused import error in __init__.py files
-	#@flake8 --ignore=F401 $(PYTHON_INIT_FILES)
-	# @echo "LINTING SUCCESSFUL"
+	pre-commit run --all-files
```

Comment:
Why was this changed? I thought it should be enough to lint only changed files.

Reply:
For those of us that don't want to install the hooks (I don't want this barrier to getting code off my machine, which apparently dies when it feels like), it doesn't capture files not in version control yet. A few of us have run into issues with not having all-files in this endpoint.

Reply:
OK, I'm fine with this change. This issue is independent of whether you have installed the git hooks - I work around it by doing `git add` on the files before I lint. At least the installed hooks still only run on updated files.

Reply:
Ok sure, though it has tripped a number of people, all-files makes it easy on the person

Reply:
Actually I think that's not true -- just ran into it again with a new file that had already been added...